var/spack/repos/builtin/packages/py-gevent/package.py

```diff
@@ -11,26 +11,36 @@ class PyGevent(PythonPackage):
 
     homepage = "https://www.gevent.org"
     pypi = "gevent/gevent-1.3a2.tar.gz"
```

Comment:
```suggestion
    pypi = "gevent/gevent-23.7.0.tar.gz"
```
Just because we're removing this version. This doesn't affect downloading, only `spack url summary`